tests/appsec/iast/source/test_cookie_value.py

```diff
@@ -20,6 +20,7 @@ class TestCookieValue(BaseSourceTest):
     def test_source_reported(self):
         super().test_source_reported()
 
+    @bug(context.weblog_variant == "jersey-grizzly2", reason="name field of source not set")
```

Comment:
With the changes of this PR, metrics tests expect to fail if the feature test is failing. I added the same `@bug` that the test already have for the test also in telemetry tests.